src/main/java/service/connection/impl/ConnectionServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package service.connection.impl;
+
+import model.connection.Connection;
+import repository.connection.ConnectionRepository;
+import repository.connection.impl.ConnectionRepositoryImpl;
+import service.connection.ConnectionService;
+
+import java.util.List;
+
+public class ConnectionServiceImpl implements ConnectionService {
+    private ConnectionRepository connectionRepository = new ConnectionRepositoryImpl();
```

Comment:
Bad practice! Read about java autowired!

Reply:
Should i create constructor like:
public ConnectionServiceImpl(ConnectionRepository repository){
        coonnectionRepository = repository;
    }
and give it ConnectionRepositoryImpl?